### PR TITLE
Agent: Add content to ransomware README.txt file

### PR DESCRIPTION
**Pull request by mssalvatore**
# What does this PR do? 

Issue #1244 

Replaces the placeholder content in the ransomware README.txt file with cool ASCII art and useful information.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing? 
* [ ] ~~Was the CHANGELOG.md updated to reflect the changes?~~
* [ ] ~~Was the documentation framework updated to reflect the changes?~~

## Testing Checklist

* [ ] ~~Added relevant unit tests?~~
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running unit tests
* [x] If applicable, add screenshots or log transcripts of the feature working

## Screenshots
image

### CODE REVIEW

**Review by shreyamalviya — APPROVED** [review]

File: monkey/infection_monkey/ransomware/ransomware_readme.txt [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+                         --.         `dNNNNNNds/:--:/sdNNNNNNd`         .--
+                                       :NNs/oydmNNNNNNNNmdyo/sNm:
+                                  .+hNN/      `.sNNs.`      /NNy+.
+                                 :my/dm.        -NN-        .md/ym-
+                                 .ddymy        `yNNy`       `ymydd.
+                                   .-.         yN//Ny         .-.
+                                               :dmmd:
+
+Infection Monkey is an open-source breach and attack simulation (BAS) platform. The files in this
+directory have been manipulated as part of a ransomware simulation. This is NOT a ransomware attack.
```
Comment (VakarisZ) [inline monkey/infection_monkey/ransomware/ransomware_readme.txt:42]: This kind of is a ransomware attack or at least we're trying to get as close as possible to one. Maybe rephrase to `This is NOT a real ransomware attack.`?

File: monkey/infection_monkey/ransomware/ransomware_readme.txt [hunk 2]
```diff
@@ -1,2 +1,49 @@
-This is a placeholder README for the Infection Monkey Ransomware Simulation.
-Don't panic :)
+ ██████████                        ██  █████       ███████████                        ███
+░░███░░░░███                      ███ ░░███       ░░███░░░░░███                      ░░░
+ ░███   ░░███  ██████  ████████  ░░░  ███████      ░███    ░███  ██████   ████████   ████   ██████
+ ░███    ░███ ███░░███░░███░░███     ░░░███░       ░██████████  ░░░░░███ ░░███░░███ ░░███  ███░░███
+ ░███    ░███░███ ░███ ░███ ░███       ░███        ░███░░░░░░    ███████  ░███ ░███  ░███ ░███ ░░░
+ ░███    ███ ░███ ░███ ░███ ░███       ░███ ███    ░███         ███░░███  ░███ ░███  ░███ ░███  ███
+ ██████████  ░░██████  ████ █████      ░░█████     █████       ░░████████ ████ █████ █████░░██████
+░░░░░░░░░░    ░░░░░░  ░░░░ ░░░░░        ░░░░░     ░░░░░         ░░░░░░░░ ░░░░ ░░░░░ ░░░░░  ░░░░░░
```
Comment (VakarisZ) [inline monkey/infection_monkey/ransomware/ransomware_readme.txt:8]: Maybe "Infection Monkey was here" would be better? In my mind, infection monkey spreading across the network is a valid reason to panic :D

**Review by VakarisZ — APPROVED** [review]

**Review by ilija-lazoroski — APPROVED** [review]

File: monkey/infection_monkey/ransomware/ransomware_readme.txt (hunk 2 above)
Comment (mssalvatore) [reply monkey/infection_monkey/ransomware/ransomware_readme.txt:8]: I'm having trouble getting "Infection Monkey was here" in a readable font. Anything easily readable comes out too large.

Since this PR has approvals, I'm going to merge it without this change. I'll have our marketing team look at this (and our documentation) before we release. They'll probably come up with something better.
